package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@articulate/hermes",
+  "version": "0.0.0",
+  "description": "Event-sourced autonomous service toolkit for Node.js",
+  "main": "src/index.js",
+  "repository": "git@github.com:articulate/hermes.git",
+  "author": "articulate",
+  "license": "MIT",
+  "scripts": {
+    "docs": "docsify serve -p 4000 ./docs",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@articulate/funky": "^1.9.1",
+    "debug": "^4.1.1",
+    "highland": "^2.13.5",
+    "mnemonist": "^0.32.0",
+    "pg": "^7.18.2",
+    "pg-query-stream": "^3.0.3",
+    "ramda": "^0.27.0",
+    "tinyfunk": "^1.7.2",
```

Comment:
Why both `ramda` and `tinyfunk`? I would have thought requiring `ramda` defeats the purpose of `tinyfunk`, namely that it's so much smaller than `ramda`.

Reply:
Originally I was just going to run with `tinyfunk`, but I needed a `once` and a `mergeDeepRight`, so I pulled in `ramda`.

I figured since it's already a dep through `@articulate/funky`, then adding it explicitly didn't cost anything.  And the function signatures and names in `tinyfunk` and `ramda` are similar but not exactly the same.  I prefer `tinyfunk` where I can use it, but I'm also a little biased.  😅

Reply:
:+1: was mostly just curious

Reply:
Funky and Ramda are both gone now.